### PR TITLE
UI for component

### PR DESCRIPTION
**Pull request by OlgVol**

### CODE REVIEW

File: src/app/action-card/action-card.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { IFeature } from '../module/feature';
+
+@Component({
+  selector: 'app-action-card',
+  templateUrl: './action-card.component.html',
+  styleUrls: ['./action-card.component.scss'],
+})
+export class ActionCardComponent {
+  showFeature = false;
+  sign = 'up';
```
Comment (irinapizane) [inline src/app/action-card/action-card.component.ts:11]: what for do you need this property?

Comment (irinapizane) [inline src/app/action-card/action-card.component.ts]: why do you need this property?

File: src/app/action-card/action-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+import { IFeature } from '../module/feature';
+
+@Component({
+  selector: 'app-action-card',
+  templateUrl: './action-card.component.html',
+  styleUrls: ['./action-card.component.scss'],
+})
+export class ActionCardComponent {
```
Comment (irinapizane) [inline src/app/action-card/action-card.component.ts:9]: you need to sort out this code. Inputs go first, then properties

File: src/app/module/feature.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export interface IFeature {
+  featureId?: number;
```
Comment (irinapizane) [inline src/app/module/feature.ts]: why do you need ID?

File: src/app/module/feature.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  featureName: string;
+  canDelete: boolean;
+  toDoList?: any;
```
Comment (irinapizane) [inline src/app/module/feature.ts]: do not use any.

File: src/app/home-page/home-page.component.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+@Component({
+  selector: 'app-home-page',
+  templateUrl: './home-page.component.html',
+  styleUrls: ['./home-page.component.scss'],
+})
+export class HomePageComponent {
+  pageTitle = 'Home Page';
+  titleColor= "red";
+  cardTitle = 'Card for Organizing Activities';
+  features: IFeature[] = [
```
Comment (irinapizane) [inline src/app/home-page/home-page.component.ts]: this data should be entered here:  [features]="features" 
in home-page.component.html

File: src/app/action-card/action-card.component.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+            data-content="There must be at least one “Must have tool” in your list. If you wish to delete this feature, you first need to create another feature with the same badge."
+            data-container="body" data-bs-custom-class="custom-popover">
+            <button class="btn" type="button" disabled *ngIf="!feature.canDelete">
+              Delete?
+            </button>
+          </span>
+          <i class="bi bi-trash" *ngIf="feature.canDelete"></i>
+        </div>
+      </li>
+      <ul class="list-group todo" *ngIf="feature.toDoList && showToDoList">
```
Comment (irinapizane) [inline src/app/action-card/action-card.component.html]: wrap unordered list in ng-container. 
it should be sth like this:
<ng-container *ngIf="feature.toDoList && showToDoList">
<ul>
// code goes here
<ul>
</ng-container>

File: src/app/module/feature.ts (hunk 3 above)
Comment (OlgVol) [reply src/app/module/feature.ts]: In feature if we need to delete only this feature, we can do it with ID. If we do not have such task, I can delete :)

**Pull request merged**